pymc3/distributions/timeseries.py

```diff
@@ -460,6 +464,72 @@ def logp(self, x):
     def _distr_parameters_for_repr(self):
         return ["mu", "cov"]
 
+    def random(self, point=None, size=None):
+        """
+        Draw random values from MvGaussianRandomWalk.
+
+        Parameters
+        ----------
+        point: dict, optional
+            Dict of variable values on which random values are to be
+            conditioned (uses default point if not specified).
+        size: int or tuple of ints, optional
+            Desired size of random sample (returns one sample if not
+            specified).
+
+        Returns
+        -------
+        array
+
+
+        Examples
+        -------
+        Create one sample from a 2-dimensional Gaussian random walk with 10 timesteps::
+
+            mu = np.array([1.0, 0.0])
```

Comment:
There's specific RST syntax for python code you should use. You can find that in other parts of the code base.

Reply:
I'm comparing my examples' formatting to `MvNormal`'s docstring's examples, and my formatting looks identical to me. Should I be looking elsewhere?